Source/Device.swift.gyb

```diff
@@ -61,14 +61,14 @@ iPads = [
             Device("iPad4",           "Device is an [iPad (4th generation)](https://support.apple.com/kb/SP662)",       "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP662/sp662_ipad-4th-gen_color.jpg",             ["iPad3,4", "iPad3,5", "iPad3,6"],            9.7,  (3, 4),   "iPad 4", 264, False, False, False, False, False, False),
             Device("iPadAir",         "Device is an [iPad Air](https://support.apple.com/kb/SP692)",                    "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP692/SP692-specs_color-mul.png",                ["iPad4,1", "iPad4,2", "iPad4,3"],            9.7,  (3, 4),   "iPad Air", 264, False, False, False, False, False, False),
             Device("iPadAir2",        "Device is an [iPad Air 2](https://support.apple.com/kb/SP708)",                  "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP708/SP708-space_gray.jpeg",                    ["iPad5,3", "iPad5,4"],                       9.7,  (3, 4),   "iPad Air 2", 264, False, False, False, False, True, False),
-            Device("iPadAir3",        "Device is an [iPad Air (3th generation)](INSERT LINK)", "https://store.storeimages.cdn-apple.com/4982/as-images.apple.com/is/image/AppleInc/aos/published/images/i/pa/ipad/mini/ipad-mini-compare-201903",                                                          [""],                                         10.5, (3, 4),   "iPad Air (3th generation)", 264, False, False, False, False, True, False),
             Device("iPad5",           "Device is an [iPad 5](https://support.apple.com/kb/SP751)",                      "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP751/ipad_5th_generation.png",                  ["iPad6,11", "iPad6,12"],                     9.7,  (3, 4),   "iPad 5", 264, False, False, False, False, True, False),
             Device("iPad6",           "Device is an [iPad 6](https://support.apple.com/kb/NotYetAvailable)",            "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP751/ipad_5th_generation.png",                  ["iPad7,5", "iPad7,6"],                       9.7,  (3, 4),   "iPad 6", 264, False, False, False, False, True, False),
+            Device("iPadAir3",        "Device is an [iPad Air (3th generation)](INSERT LINK)",                          "https://store.storeimages.cdn-apple.com/4982/as-images.apple.com/is/image/AppleInc/aos/published/images/i/pa/ipad/mini/ipad-mini-compare-201903", ["iPad11,3", "iPad11,4"],       10.5, (3, 4),   "iPad Air (3th generation)", 264, False, False, False, False, True, False),
```

Comment:
Nitpick: 3rd generation

Reply:
Yeah, `.iPad3` is called iPad (3th generation) so I think we should leave it like this?

Reply:
Sorry for being unclear 🙈 I meant the text (comment) “Device is an [iPad Air 3**rd** generation)]”, not “3**th** generation.

Reply:
Ah yes, I will fix that 🤣